src/components/SignupsSplit.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useTreatments, SplitContext } from '@splitsoftware/splitio-react';
+
+const SignupsSplit = ({
+  testChildren,
+  defaultChildren,
+  treatmentWithConfig,
+}) => {
+  const treatments = useTreatments(['signups']);
+  const { treatment } = treatmentWithConfig;
+  const { isReady } = React.useContext(SplitContext);
+
+  return isReady && treatment !== 'off' && treatments === 'signups' ? (
```

Comment:
`treatments` should be an object at this time, so I'm not sure this will ever be true. Adding a check for the 'control' treatment should be fine. That's when things haven't quite loaded yet.
```suggestion
  return isReady && treatment !== 'off' && treatment === 'control' ? (
```